Generator/TypeGenerator.php

```diff
@@ -216,8 +216,14 @@ public function loadClasses($forceReload = false)
     {
         if (!self::$classMapLoaded || $forceReload) {
             $classes = require $this->getClassesMap();
-
-            $mapClassLoader = new ClassLoader();
+            /** @var ClassLoader $mapClassLoader */
+            static $mapClassLoader = null;
+            if (null === $mapClassLoader) {
+                $mapClassLoader = new ClassLoader();
+                $mapClassLoader->setClassMapAuthoritative(true);
+            } else {
```

Comment:
Is it possible to enter this else statement? `$mapClassLoader` seems to be always null to me or am I missing something?

Reply:
No it is null on init because $mapClassLoader is declare as static so after it init it always has the same instance of ClassLoader.